packages/cashier/src/types/stores/common-store.types.ts

```diff
@@ -1,4 +1,5 @@
 export type TCommonStore = {
     platform: string;
+    is_from_derivgo: boolean;
```

Comment:
@vinu-deriv please add `is_from_derivgo` in alphabetic order to minimise conflicts in the future 🙏🏻